.travis.yml

```diff
@@ -12,6 +12,9 @@ matrix:
   include:
     - os: linux
       addons:
+        hosts:
+          - jsscbuild
+        hostname: jsscbuild
```

Comment:
Can you please explain what problem this hosts/hostname solves?

Reply:
Answering my own question... this seems to be derived from a 2015 post from this thread: https://github.com/travis-ci/travis-ci/issues/5227#issuecomment-165131913.  I believe the correlation between these issues is presumptuous and that Travis-CI should be eliminated for proper troubleshooting.  Specifically, I can reproduce the Buffer Overflow on a 14.04 machine running Oracle Java.

Reply:
I agree, this didn't help. Let's remove this again.